querybook/webapp/components/QuerySnippetInsertionModal/QuerySnippetView.tsx

```diff
@@ -68,7 +68,9 @@ export class QuerySnippetView extends React.PureComponent<
         if (templatedVariables.length) {
             renderedQuery = await renderTemplatedQuery(
                 context,
-                this.state.templatedQueryForm,
+                Object.entries(this.state.templatedQueryForm).map(
```

Comment:
curious why we need to do `renderTemplatedQuery` when inserting snippet? shouldn't the variables be kept in the query, insert as is?

Reply:
when inserting the snippets, the user needs to fill out the template form